gobblin-runtime/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -367,6 +381,24 @@ private void scheduleLocallyConfiguredJobs() throws ConfigurationException, JobE
   }
 
   /**
+   * Load job configuration file(s) from general source
+   */
+  private List<Properties> loadGeneralJobConfigs() throws ConfigurationException, IOException {
+    List<Properties> jobConfigs = SchedulerUtils.loadGenericJobConfigs(this.properties);
+    LOG.info(String.format(jobConfigs.size() <= 1 ? "Loaded %d job configuration" : "Loaded %d job configurations",
+        jobConfigs.size()));
+    return jobConfigs ;
+  }
+
+  /**
+   * TODO: Support general file system API for configure file monitor.
+   * Job configuration file monitor using generic file system API.
+   */
+  private void startGeneralJobConfigFIleMonitor() throws Exception {
+    return ;
```

Comment:
For void methods, omit the `return` statement. Also, consider using `UnsupportedOperationException`.